package.json

```diff
@@ -52,7 +52,7 @@
     "prebuild": "yarn typechain"
   },
   "dependencies": {
-    "@nucypher/nucypher-core": "^0.11.0",
+    "@nucypher/nucypher-core": "file:../nucypher-core/nucypher-core-wasm/pkg",
```

Comment:
I suppose this line will be changed when https://github.com/nucypher/nucypher-core/pull/77 is merged, right?

Reply:
Yes, but specifically when we release `@nucypher/nucypher-core@0.12.0` on npm.